forms-shared/src/schemas/tsb/shared/extractInfoShared.ts

```diff
@@ -0,0 +1,21 @@
+import { GenericObjectType } from "@rjsf/utils";
+import { safeString } from "../../../form-utils/safeData";
+
+export const tsbExtractEmail = (formData: GenericObjectType) => {
+  return safeString(formData.objednavatel?.email || formData.ziadatel?.email)
+}
+
+export const tsbExtractName = (formData: GenericObjectType) => {
+  if (formData.objednavatel) {
+    if (formData.objednavatel.objednavatelTyp === 'Fyzická osoba' || formData.objednavatel.objednavatelTyp === 'Fyzická osoba - podnikateľ') {
```

Comment:
I would probably change it to camel cased enums like `fyzickaOsoba` in the future in form defintion. However I will edit it then.